packages/tools/fluid-runner/README.md

```diff
@@ -39,16 +39,17 @@ The default format is currently `JSON`
 There is an optional command line option `telemetryProp` that allows you to specify additional properties that will be added to every telemetry entry. The format follows these rules:
 - every key must be a string
 - values may be either a string or a number
-- a value may not be empty
+- keys and values cannot be empty
 
 Example of valid usages:
 ```
 --telemetryProp prop1 value1 --telemetryProp prop2 10.5
 --telemetryProp "  prop1 " "   value1 " prop2 value2
```

Comment:
I'd not do trimming, unless we know that spaces are not allowed.

Reply:
There is no trimming performed. Empty is referring to empty string/undefined. Whitespace is allowed